zone/command.cpp

```diff
@@ -387,6 +387,7 @@ int command_init(void)
 		command_add("unscribespells", "- Clear out your or your player target's spell book.", AccountStatus::GMCoder, command_unscribespells) ||
 		command_add("untraindisc", "[spellid] - Untrain specified discipline from your target.", AccountStatus::GMCoder, command_untraindisc) ||
 		command_add("untraindiscs", "- Untrains all disciplines from your target.", AccountStatus::GMCoder, command_untraindiscs) ||
+		command_add("updatechecksum", "gm only", AccountStatus::GMImpossible, command_updatechecksum) ||
```

Comment:
Could use a bit more verbosity here on the command note.

Reply:
Yeah I didn't look at this when Im resurrected it - made it more descriptive.